x-pack/packages/kbn-elastic-assistant/impl/assistant_context/index.tsx

```diff
@@ -45,21 +45,29 @@ type ShowAssistantOverlay = ({
 interface AssistantProviderProps {
   actionTypeRegistry: ActionTypeRegistryContract;
   augmentMessageCodeBlocks: (currentConversation: Conversation) => CodeBlockDetails[][];
+  baseAllow: string[];
+  baseAllowReplacement: string[];
+  defaultAllow: string[];
+  defaultAllowReplacement: string[];
```

Comment:
Maybe s/defaultAllowReplacement/defaultAllowAnonymization/g as we discussed? 🤷‍♂️ For another day 🙂

Reply:
Also maybe `base` <-> `default`? This is my fault since I named the `basePromptContexts` and `baseQuickPrompts`, sorry! 😬